src/PageCache/RequestPolicy/Cookie.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Drupal\ssofact\PageCache\RequestPolicy;
+
+use Drupal\Core\PageCache\RequestPolicyInterface;
+use Symfony\Component\HttpFoundation\Request;
+
+/**
+ * Reject when the user has the sso cookie.
+ */
+class Cookie implements RequestPolicyInterface {
```

Comment:
1. I may not be up to date with the class naming conventions in D8.  Shouldn't the class have a suffix to clarify what it is?  I.e., `CookieRequestPolicy`?

2. In any case, the name could be more concise as we're not disallowing caching on _any_ kind of cookie; e.g., `SessionCookieRequestPolicy`?